Source/WPF/MyMoney/Views/GraphGenerators.cs

```diff
@@ -73,14 +77,14 @@ public IEnumerable<TrendValue> Generate()
                             Value = balance,
                             UserData = t
                         };
-                    }
+                    }                    
                 }
             }
         }
 
         public string GetLabel(TrendValue item)
         {
-            return item.Value.ToString("n", nfi) + "\r\n" + item.Date.ToShortDateString();
+            return item.Value.ToString("n", nfi);

```

Comment:
I'm sure this is correct, I saw the other place where you moved the date.
I'm probably not looking deep enough, but it looks like we are dumping the information about date here.